src/Frontend/Themes/Bootstrap/Modules/Blog/Layout/Templates/Category.html.twig

```diff
@@ -0,0 +1,59 @@
+{#
+variables that are available:
+- {{ category }}: contains data about the category
+- {{ items }}: contains an array with all posts, each element contains data about the post
+#}
+
+{% if items %}
+<section id="blogCategory" class="blog">
```

Comment:
is het de bedoeling dat er geen indentatie is in de twig tags? De twig coding standards zeggen dit daarover:

> Indent your code inside tags (use the same indentation as the one used for the target language of the rendered template):

http://twig.sensiolabs.org/doc/coding_standards.html